### PR TITLE
sphinx-autobuild and README instructions

### PR DESCRIPTION
**Pull request by pauleveritt**
For easier authoring, add in sphinx-autobuild. Put some minimal dev info in the repo README.

### CODE REVIEW

File: README.md
Comment (CAM-Gerlach) [inline README.md:1]: The title is "Python Docs WG", but the full name below is given as the "PSF Documentation Working Group", and the shortname as "Docs WG". Can we be consistant with the fullname and shortname, and use either one or the other here?

File: README.md
Comment (CAM-Gerlach) [inline README.md:5]: Does it make sense to include "Python" in the full name, since it is specifically for the documentation for the Python language (though I suppose that is what the P in PSF stands for)?

Repetition of PSF without defining what it stands for, and since this repo is under the Python org, not the PSF one, it would make sense to link the [PSF](https://www.python.org/psf/) page.

```suggestion
The PSF Documentation Working Group (Docs WG) was chartered by the [Python Software Foundation](https://www.python.org/psf/) to help bootstrap a larger documentation team and community.
```

File: README.md
Comment (CAM-Gerlach) [inline README.md:8]: I'm not sure how "finding out more about us" is an "example"? I find this rather confusing, which is likely more so for many non-native speakers—I've found out firsthand that humor, irony and cultural references don't always translate well, and style guides for docs (including my own) tend to proscribe it for that reason.

File: README.md
Comment (CAM-Gerlach) [inline README.md:15]:
~~~suggestion
## Build docs and view changes

1. Clone this repo

   ```console
   git clone https://github.com/python/docs-community.git
   cd docs-community
   ```

2. Create a new Python virtual environment

   ```console
   python -m venv docs-wg-env
   ```

3. Activate the environment ([platform/shell-specific](https://docs.python.org/3/tutorial/venv.html#creating-virtual-environments))

4. Install dependencies

   ```console
   python -m pip install -r requirements.txt
   ```
5. Build the docs, open them in your browser and update whenever changes are made

   ```console
   sphinx-autobuild --open-browser docs docs/_build
   ```
~~~

Make this section much clearer, more descriptive and more useful

**Review by CAM-Gerlach — COMMENTED** [review]
One Sentence Per Line is the One True Way :+1: :+1: :+1: 

I'm not sure how necessary adding `sphinx-autobuild` as a hard requirement is here, but eh.

**Review by AA-Turner — APPROVED** [review]
Looks good, generally agree with Cam's comments

A

File: README.md
Comment (CAM-Gerlach) [reply README.md:15]: I've always struggled with this with venv (since like you, most of my work uses conda envs which by design use descriptive names, whereas venv names are all over the place—some use descriptive names, while some use one of perhaps a half dozen different "standard names", e.g. `venv`, `env`, `.venv`, `.env`, etc). The docs don't formally recommend anything, some venv/virtualenv-based systems (`virtualenvwrapper`, `virtualenvhelper`, `mkvirtualenv`, etc, etc.) do use centralized names, and the name gets injected into the prompt, where it is helpful to be descriptive to confirm the user is in the venv they expect (as I've made that mistake before).

As such, I went with a descriptive name here, but I'm happy to defer to something else if someone who knows better than me (which likely includes most if not all of you) thinks so.

**Comment by pauleveritt** [comment]
I'll confess, this commit was just a very quick README. There was nothing there before, this repo is mostly for us internally, and major content work is coming in future commits. This one was really about the tiny act of getting `sphinx-autobuild` included.

I will include Cam's section and fix up the "what is our name" based on Marietta's statement.

File: README.md
Comment (encukou) [reply README.md:1]: Technically, the WG is a formal group of ~four people, who can vote and do other formal things.
Arguably the group isn't working all that well, and IMO it should only be invoked when we need something like a formal pronouncement on some drawn-out argument.

"Python Documentation Community" (or whatever we'll call ourselves) is the wider group of basically anyone who wants to help (including members of the Docs WG). That should be the main focus of this repo.

File: requirements.txt
Comment (encukou) [inline requirements.txt:4]: sphinx-copybutton

**Comment by encukou** [comment]
Hm, I tried GitHub's online conflict resolution, and made more noise than what I wanted.

Anyway, anyone against merging this now?

File: README.md
Comment (AA-Turner) [inline README.md:1]:
```suggestion
# Python Documentation Community
```

**Comment by AA-Turner** [comment]
> anyone against merging this now?

There are two unresolved items, the "community" versus "working group" framing, and the longer instructions.

Those are very minor though so can be done later, Paul mentioned the point of this was to add the autobuild functionality.

A

**Comment by CAM-Gerlach** [comment]
@encukou It seems like @pauleveritt intended to apply my suggestions and the one you did, based on [his comment above](https://github.com/python/docs-community/pull/25#issuecomment-1046011028), but never got to doing so (or otherwise modifying this PR). Therefore, I would suggest applying them before merging, as it seems @pauleveritt intended, given they are a fairly substantial improvement to the usefulness of the document (which I wouldn't call "very minor"), and if the PR is merged, they would have to each be manually redone which is rather wasteful.

**Comment by encukou** [comment]
Yup, that seems best. Things can always be fixed up in later PRs.

**Pull request merged**
